rocAL/rocAL/source/augmentations/geometry_augmentations/node_crop.cpp

```diff
@@ -77,12 +77,9 @@ void CropNode::init(unsigned int crop_h, unsigned int crop_w)
 {
     _crop_param->crop_w = crop_w;
     _crop_param->crop_h = crop_h;
-    _crop_param->x1     = 0;
-    _crop_param->y1     = 0;
-    FloatParam *x_drift  = ParameterFactory::instance()->create_single_value_float_param(0.5);
-    FloatParam *y_drift  = ParameterFactory::instance()->create_single_value_float_param(0.5);
-    _crop_param->set_x_drift_factor(core(x_drift));
-    _crop_param->set_y_drift_factor(core(y_drift));
+    _crop_param->x1 = 0; 
+    _crop_param->y1 = 0;
```

Comment:
I hope this particular init is used only for center crop, The previous use case is different, ensure it does not affect any other use cases which utilizes this init

Reply:
@swetha097 Better add a comment stating this will be called only for centre crop.